src/AddExerciseForm.tsx

```diff
@@ -27,8 +28,9 @@ const AddExerciseForm: FC<ExerciseFormProps> = ({ selectedDay, handleClose }: Ex
       sets,
       reps,
       weight,
+      done: false,
     };
-    if (name !== '' && handleClose) {
+    if (name !== '') {
```

Comment:
Try to replace the contents of this if statement with `event.target.validity`. Should work.